src/modules/Builder/features/ActivitySettings/ActivitySettings.tsx

```diff
@@ -1,69 +1,41 @@
-import { useState, useEffect } from 'react';
 import { useNavigate, useParams, generatePath } from 'react-router-dom';
-import { useTranslation } from 'react-i18next';
 
 import { page } from 'resources';
 import { useBreadcrumbs } from 'shared/hooks';
 import { useActivitiesRedirection, useCurrentActivity } from 'modules/Builder/hooks';
+import { NavigationMenu } from 'shared/components';
+import { NavigationItem } from 'shared/components/NavigationMenu/NavigationMenu.types';
```

Comment:
could you pls export NavigationMenu.types from index to import NavigationItem with shorter path?